shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -362,12 +362,13 @@
 }
 
 bool FlutterPlatformViewsController::SubmitFrame(GrContext* gr_context,
-                                                 std::shared_ptr<IOSGLContext> gl_context) {
+                                                 fml::WeakPtr<IOSGLContext> onscreen_gl_context,
+                                                 fml::WeakPtr<IOSGLContext> resource_gl_context) {
```

Comment:
ditto

Reply:
this is a C++ class, so {resource,onscreen}_gl_context is correct